Source/WebKit/UIProcess/WebPageProxy.cpp

```diff
@@ -460,6 +460,42 @@ Ref<WebPageProxy> WebPageProxy::create(PageClient& pageClient, WebProcessProxy&
     return adoptRef(*new WebPageProxy(pageClient, process, WTFMove(configuration)));
 }
 
+WebPageProxy::ProcessActivityState::ProcessActivityState(WebPageProxy& page)
+    : m_page(page)
+    , m_wasRecentlyVisibleActivity(makeUniqueRef<ProcessThrottlerTimedActivity>(8_min))
```

Comment:
`#if PLATFORM(MAC)`

I think here we should make it clear that this behavior is platform-specific.